src/sagas/helpers.js

```diff
@@ -229,3 +229,17 @@ export function disableFeaturesIfNeeded(customNetworkSettings, currentFeatureTog
   }
   return { ...featureToggles };
 }
+
+/**
+ * Retrieve the networkSettings either from storage or the state.
+ * @param {object} state being the redux state containing the current networkSettings.
+ */
+export function getNetworkSettings(state) {
+  // The state is always present, but the stored network settings
```

Comment:
The state is not always present, right? When you close the app, it loses the state data but keeps the storage. When you start the app again, you will have different values for the storage data and redux data. Are you copying the storage data to the redux when the app is started?

Reply:
Oh! You right, it is lacking an updated in the state when the wallet initializes, as it happens to pushNotification. Let me update it.

Reply:
- Add initNetworkSettings effect: 2e3de2987dbdff68cbc7667dd7cce4c6edf98848